rsts/_static/algolia.js

```diff
@@ -0,0 +1,6 @@
+docsearch({
+apiKey: 'af11c512aa92422933b48828d28dbe5f',
+indexName: 'flyte',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/flyteorg/flyte/pullRequest?prid=7095401)
```suggestion
indexName: "flyte",
```